### PR TITLE
Full app

### PR DESCRIPTION
**Pull request by brytebee**
## React Routing 🌍💻 
In the PR, I have achieved the following:
- added multiple pages to the web app
- installed and implemented react routing
- added navbar and footer
- styled newly added components

### CODE REVIEW

**Review by mikenath223 — CHANGES_REQUESTED** [review]
Hello @brytebee 

You've done pretty well so far!
But for a few issues, you are almost done with this. Hang in there!


## Required Changes ♻️
_Please check the comments under the review._


Happy coding, Cheers! 🚀🚀

Do feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, whether it is valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/App.js [hunk 1]
```diff
@@ -1,5 +1,21 @@
...
+import Quote from './pages/Quote';
 
-const App = () => <Calculator />;
+const App = () => (
+  <>
+    <Nav />
+    <Routes>
+      <Route path="/" exact element={<Home />} />
+      <Route path="/quote" element={<Quote />} />
+      <Route path="/calculator" element={<Calculator />} />
```
Comment (mikenath223) [inline src/App.js]: - [ ] Kindly deploy the project online and include the live link in the readme.

**Comment by D4nielJ** [comment]
## Hello @brytebee, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear or you can always find me on slack!

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by brytebee** [comment]
Hello, @D4nielJ 👋👋,

Thanks for the warm-hearted review 👀👀.



I'll go ahead to merge and move on to the next milestone 🏁🏁.

**_Happy Hacking 😄😄_**

**Pull request merged**